### DISCUSSION TITLE
Swagger com JSON faltando campo

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#632] [Tira Dúvidas] [answered]
Pessoal,
Tenho uma request que está assim:
public class LinkUsuarioEmpresaEscritorioRequest
{
    public string UsuarioEmail { get; } = string.Empty;        
    public string EmpresaCnpj { get; set; } = string.Empty;
    public string EscritorioCnpj { get; set; } = string.Empty;
}
e é ela que uso no Handler e no Endpoint....
em nenhum momento eu usei o JSONIGNORE

mas quando o swagger gera a documentação o campo UsuarioEmail não aparece

![image](https://github.com/user-attachments/assets/0838504c-bb3e-4c28-b494-7ab4d2d81101)

devo ter feito algo errado mas não to conseguindo encontrar....

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @FBODeveloper, como vai?
Tem certeza que está no request certo?

**Reply by FBODeveloper** [reply]
Bom dia Balta....bem graças a Deus... e por aí?
Cara erro de principiante...kkk...
olha minha classe acima.... faltou o set; no UsuarioEmail....
nem sei quantas horas perdi nisso kkk
Valeu

**Reply by andrebaltieri** [reply]
hahahaha acontece!!!
